django-verdant/rca/static/rca/js/showcase-block.js

```diff
@@ -0,0 +1,26 @@
+$(function(){
+    // Showcase carousel
+    var $showcaseSlider = $('.js-showcase');
+    var prevButton = '<button aria-label="Previous" class="showcase__prev showcase__button" type="button"><svg class="showcase__arrow"><title>Previous</title><use xlink:href="#chevron"></use></svg></button>';
+    var nextButton = '<button aria-label="Next" class="showcase__next showcase__button" type="button"><svg class="showcase__arrow"><title>Previous</title><use xlink:href="#chevron"></use></svg></button>';
```

Comment:
in my branch I made some adjustments to this markup to make them more generic so I could reuse the buttons and styles on the testimonial block. We should probably discuss the best approach with these. I'll split the js in my branch out into separate files to match what you've done here

Reply:
thanks and yes let's have a chat - will check your testimonial branch next.